### PR TITLE
libnl: update to 3.5.0

### PR DESCRIPTION
**Pull request by dengqf6**
Tested on mvebu

### CODE REVIEW

**Comment by ynezz** [comment]
@dengqf6 you're missing commit description

File: package/libs/libnl/patches/101-add-musl-workaround-to-the-libc-compat.h-copy.patch [hunk 1]
```diff
@@ -1,51 +1,35 @@
 --- a/include/linux-private/linux/if_ether.h
```
Comment (hauke) [inline package/libs/libnl/patches/101-add-musl-workaround-to-the-libc-compat.h-copy.patch]: This patch should not be needed any more, this upstream commit should fix the problem with musl libc:
https://github.com/thom311/libnl/commit/73c1d0479643cf743b72879084e70f738f11bded

File: package/libs/libnl/patches/102-revert-build-enable-building-cli-during-tests.patch [hunk 2]
```diff
@@ -37,18 +37,18 @@ Upstream status: https://github.com/thom311/libnl/pull/141
```
Comment (hauke) [inline package/libs/libnl/patches/102-revert-build-enable-building-cli-during-tests.patch]: The linked pull request mentions that this problem was also fixed some time ago:
 https://github.com/thom311/libnl/pull/141
This is probably also not needed any more

**Comment by hauke** [comment]
Thank you for the patch, it was applied to master.

**Pull request closed without merging**